### PR TITLE
Include staking auction length in `resetEpoch`

### PR DESCRIPTION
**Pull request by jordanschalm**
This PR updates the transaction argument generation tool for `resetEpoch` to include the `stakingEndView` parameter added in https://github.com/onflow/flow-core-contracts/pull/268. 

⚠️ Before merging, tag a release for `core-contracts` including https://github.com/onflow/flow-core-contracts/pull/267.

### CODE REVIEW

**Review by kc1116 — APPROVED** [review]
Approved

File: cmd/util/cmd/epochs/cmd/reset.go [hunk 1]
```diff
@@ -120,19 +121,57 @@ func extractResetEpochArgs(snapshot *inmem.Snapshot) []cadence.Value {
...
 	finalView, err := epoch.FinalView()
 	if err != nil {
 		log.Fatal().Err(err).Msg("could not get final view from epoch")
 	}
 
-	return convertResetEpochArgs(epochCounter, randomSource, flagPayout, firstView, finalView)
+	return convertResetEpochArgs(epochCounter, randomSource, flagPayout, firstView, stakingEndView, finalView)
+}
+
+// getStakingAuctionEndView determines the staking auction end view from the
```
Comment (vishalchangrani) [inline cmd/util/cmd/epochs/cmd/reset.go:139]: ❤️ thank you for this explanation

**Review by vishalchangrani — APPROVED** [review]

**Comment by jordanschalm** [comment]
bors merge

**Comment by jordanschalm** [comment]
bors merge

**Comment by jordanschalm** [comment]
bors merge

**Pull request merged**